### PR TITLE
Task 305-311: QUAREL and deal_or_no_dialog datasets

### PR DESCRIPTION
**Pull request by jacob0527**

### CODE REVIEW

**Comment by swarooprm** [comment]
@kuntalkumarpal, wondering if you can review this PR?

File: tasks/README.md [hunk 1]
```diff
@@ -6,4 +6,11 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task305_QUAREL_correct_answer_generation` | Given a sentence and a question, write the correct answer based on the sentence. | Answer Generation
+`task306_QUAREL_incorrect_answer_generation` | Given a sentence and a question, write the incorrect correct answer based on the sentence. | Incorrect Answer Generation
```
Comment (kuntalkumarpal) [inline tasks/README.md:11]: incorrect correct means ?

**Comment by kuntalkumarpal** [comment]
For 
Rephrase "The input is taken from a negotiation between two participants, output Yes if both participants agree to the deal, otherwise output No." to Given a negotiation between two participants, answer Yes if both participants agree to the deal, otherwise answer No.

**Comment by kuntalkumarpal** [comment]
For 311
Give one postive example for No

**Comment by kuntalkumarpal** [comment]
@swarooprm I think task 305 and 307 , 306 and 308 are same. Asking to generate options , asking to generate option number. Do you accept them as separate tasks ?

File: tasks/README.md [hunk 2]
```diff
@@ -6,4 +6,11 @@ This directory contains the tasks that are part of this benchmark.
...
+`task307_QUAREL_correct_option_generation` | Given a sentence and a question, choose the correct option based on the sentence. | Answer Generation
+`task308_QUAREL_incorrect_option_generation` | Given a sentence and a question, choose the incorrect option based on the sentence. | Incorrect Answer Generation
```
Comment (kuntalkumarpal) [inline tasks/README.md:13]: "Choose the option number instead of exact answer". Else it looks same

File: tasks/README.md [hunk 3]
```diff
@@ -6,4 +6,11 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task305_QUAREL_correct_answer_generation` | Given a sentence and a question, write the correct answer based on the sentence. | Answer Generation
```
Comment (kuntalkumarpal) [inline tasks/README.md:10]: Also lower case the filenames and also in the name column in README. `task305_QUAREL_correct_answer_generation` to task305_quarel_correct_answer_generation

Comment (jacob0527) [reply tasks/README.md]: done

File: tasks/README.md (hunk 2 above)
Comment (jacob0527) [reply tasks/README.md]: done

File: tasks/README.md (hunk 1 above)
Comment (jacob0527) [reply tasks/README.md]: typo, just corrected it

File: tasks/README.md (hunk 3 above)
Comment (jacob0527) [reply tasks/README.md]: Screen Shot 2021-09-14 at 7 34 52 PM

Hi, I've renamed the filenames, but it showed "all checks have failed" when I committed the changes. Does it have any impact on the changes?

**Comment by kuntalkumarpal** [comment]
@swarooprm  looks good. Do you have any comments ?

**Comment by swarooprm** [comment]
> @swarooprm I think task 305 and 307 , 306 and 308 are same. Asking to generate options , asking to generate option number. Do you accept them as separate tasks ?

No, they are not unique. But @jacob0527 has extra tasks and we evaluate best 4

**Comment by swarooprm** [comment]
Your HW2 is complete
Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo (which at this point is 442) and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this @jacob0527 ?

**Comment by swarooprm** [comment]
@jacob0527  Just a reminder that, if you plan to submit these tasks to the main Github repo and join our research project, Monday (27th Sept) is the deadline. Again, this is optional and independent of your HW2 grade.

**Pull request closed without merging**